### PR TITLE
scholarly-epfl-thesis:0.1.0

### PR DESCRIPTION
**Pull request by augustebaum**
I am submitting
- [x] a new package
- [ ] an update for a package

Description: A template for a thesis at EPFL.

I have read and followed the submission guidelines and, in particular, I
- [x] selected a name that isn't the most obvious or canonical name for what the package does
- [x] added a `typst.toml` file with all required keys
- [x] added a `README.md` with documentation for my package
- [x] have chosen a license and added a `LICENSE` file or linked one in my `README.md`
- [x] tested my package locally on my system and it worked
- [x] `exclude`d PDFs or README images, if any, but not the LICENSE

- [x] ensured that my package is licensed such that users can use and distribute the contents of its template directory without restriction, after modifying them through normal use.

### CODE REVIEW

**Comment by laurmaedje** [comment]
There are by now various templates starting with "modern-". We think that the profileration of this term isn't great for users and not really in the spirit of the naming rules. While it's not forbidden, we strongly discourage its use. Instead, we recommend you choose an adjective that more closely relates to your template.

**Comment by augustebaum** [comment]
> ... we recommend you choose an adjective that more closely relates to your template.

How about "scholarly"?

**Comment by laurmaedje** [comment]
That would work.

File: packages/preview/scholarly-epfl-thesis/0.1.0/typst.toml [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+version = "0.1.0"
+compiler = "0.11.0"
+entrypoint = "src/lib.typ"
+repository = "https://github.com/augustebaum/epfl-thesis-typst"
+authors = ["Auguste Baum <https://github.com/augustebaum>"]
+license = "MIT"
+description = "A template for a thesis at EPFL"
+keywords = ["thesis", "university", "EPFL", "Lausanne"]
+categories = ["thesis"]
+exclude = ["example", "latex-reference", "screenshots"]
```
Comment (elegaanz) [inline packages/preview/scholarly-epfl-thesis/0.1.0/typst.toml:12]: I think `example` should not be excluded here, as it is the directory that is used to initialize the template.

Comment (augustebaum) [reply packages/preview/scholarly-epfl-thesis/0.1.0/typst.toml]: Thanks for your review! I misunderstood the point of `exclude` I guess. This is now corrected.

Comment (elegaanz) [reply packages/preview/scholarly-epfl-thesis/0.1.0/typst.toml]: `exclude` tells which files should not be in the archive that will be downloaded by Typst when your package is imported. It is generally a good practice to exclude example files, but in your case you also use this directory to initialize the template associated with your package, which would not work if the directory is not part of the downloaded archive.

**Pull request merged**